src/mmg3d/mmg3d1.c

```diff
@@ -964,14 +988,18 @@ static int MMG5_coltet(MMG5_pMesh mesh,MMG5_pSol met,int8_t typchk) {
               else {
                 if ( mesh->adja[4*(k-1)+1+i] )  continue;
                 if (MMG5_boulesurfvolpNom(mesh,k,ip,i,
-                                          list,&ilist,lists,&ilists,&refmin,&refplus,p0->tag & MG_NOM) < 0 )
+                                          list,&ilist,lists,&ilists,&refmin,&refplus,p0->tag & MG_NOM) < 0 ){
+                  MMG5_show_tet_location(mesh, pt, k);
                   return -1;
+                }
               }
             }
             else {
               if (MMG5_boulesurfvolp(mesh,k,ip,i,
-                                     list,&ilist,lists,&ilists,p0->tag & MG_NOM) < 0 )
+                                     list,&ilist,lists,&ilists,p0->tag & MG_NOM) < 0 ){
+                MMG5_show_tet_location(mesh, pt, k);
```

Comment:
Please, keep this in debug mode only (`-d` command line option, `mesh->info.ddebug` variable inside the code) : 
```
if ( mesh->info.ddebug ) {
   MMG5_show_tet_location(mesh, pt, k);
}
```